template/main.typ

```diff
@@ -53,12 +53,12 @@ These functions will overwrite the standard #link("https://typst.app/docs/refere
 Here is how Typst's own #std-smallcaps[smallcaps] and #std-upper[upper] look compared to the 'Ilm's variants:\
 #hide[Here is how Typst's own ] #smallcaps[smallcaps] and #upper[upper]
 
-They both look similar, the only difference is that 'Ilm uses more spacing between individual characters.
+They both look similar, the only difference being that 'Ilm uses more spacing between individual characters.
 
-If you prefer Typst's default spacing then you can still use it by prefixing `std-` to the functions: ```typst #std-smallcaps()``` and ```typst #std-upper()```.
+If you prefer Typst's default spacing, you can still use it by prefixing `std-` to the functions: ```typst #std-smallcaps()``` and ```typst #std-upper()```.
```

Comment:
I believe, feel free to correct me if I'm mistaken, this change is not due to a grammatical error but rather subjective.

I like how the sentence reads with `then`. Kindly revert this change.